text/0000-prepare-phase.md

```diff
@@ -0,0 +1,105 @@
+# Meta
+[meta]: #meta
+- Name: Prepare Phase
+- Start Date: 2021-01-13
+- Author(s): [@jkutner](github.com/jkutner/)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: [buildpacks/spec#176](https://github.com/buildpacks/spec/pull/176)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new Lifecycle phase, called "prepare", that would run before all other phases.
+
+# Definitions
+[definitions]: #definitions
+
+* __stack descriptor__ - an enhanced version of the [`stack.toml`](https://github.com/buildpacks/spec/blob/main/platform.md#stacktoml-toml)
+* __project descriptor__ - the [`project.toml`](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md) extension specification
+
+# Motivation
+[motivation]: #motivation
+
+The prepare phase would support the following features and capabilities:
+* [Stack buildpacks](https://github.com/buildpacks/rfcs/pull/111), which require a phase to read run-image mixins validation prior to detection
+* [Inline buildpacks](https://github.com/buildpacks/rfcs/blob/main/text/0048-inline-buildpack.md), which require parsing of the `project.toml` in the lifecycle
```

Comment:
It might be a good idea to link to the pack PR that added support for inline buildpacks. I expect that pack may eventually remove this code in favor of running the prepare phase.

Reply:
i added like **The prepare phase would remove this features from pack:** then i added the pr link.